### ISSUE TITLE
childproc.spawn is not a function

### ISSUE THREAD

**Issue by Darahat** [shime/dominant-color#6]
when i run my programme it showing this error

> imagemagick.js?d8c9:24 Uncaught TypeError: childproc.spawn is not a function
    at exec2 (imagemagick.js?d8c9:24)
    at Object.exports.convert (imagemagick.js?d8c9:253)
    at module.exports (index.js?d311:14)
    at eval (Header.vue?5e07:28)
    at Module../node_modules/cache-loader/dist/cjs.js?!./node_modules/babel-loader/lib/index.js!./node_modules/cache-loader/dist/cjs.js?!./node_modules/vue-loader/lib/index.js?!./src/components/common/Header.vue?vue&type=script&lang=js& (app.js:1072)
    at __webpack_require__ (app.js:725)
    at fn (app.js:102)
    at eval (Header.vue?6677:1)
    at Module../src/components/common/Header.vue?vue&type=script&lang=js& (app.js:7627)
    at __webpack_require__ (app.js:725)

**Comment by bswank** [comment]
@Darahat did you ever get this figured out? Trying to use this package now and I'm running into the same issue.

**Comment by Darahat** [comment]
@bswank  I cant figure out this problem.if u want u can use 
Color theif: https://github.com/lokesh/color-thief/
Or
Vibrant.js:
https://github.com/jariz/vibrant.js/

I used color-thief for that time.

**Comment by ntatko** [comment]
I don't think this module is set up to run inside webpack. It's probably possible to build a loader or plugin.
